client/src/components/StudentMentorship.js

```diff
@@ -365,7 +398,7 @@ class SchedulingCards extends Component {
                         <Button onClick={this.toggleFeedbackModal.bind(this)}>
                             Done
                         </Button>
-                        <Button onClick={this.submitFinalNote.bind(this)} primary>
+                        <Button onClick={this.submitFinalNote.bind(this)} primary disabled={!this.state.publicFeedback || !this.state.privateFeedback}>
```

Comment:
Students should not be able to submit without having public or private feedback for the alumni
Testimonials are more special and doesn't have an immediate impact on alumni, and thus should only be submitted when other feedback is given

Reply:
Makes sense! let's go with it